partnercentersellv1/partner_center_sell_v1_examples_test.go

```diff
@@ -0,0 +1,1004 @@
+			// begin-create_registration
+
+			primaryContactModel := &partnercentersellv1.PrimaryContact{
+				Name:  core.StringPtr("Petra"),
+				Email: core.StringPtr("petra@ibm.com"),
+			}
+
+			createRegistrationOptions := partnerCenterSellService.NewCreateRegistrationOptions(
+				accountId,
+				"Sample_company_sdk",
+				primaryContactModel,
+			)
+
+			registration, response, err := partnerCenterSellService.CreateRegistration(createRegistrationOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(registration, "", "  ")
+			registrationId = *registration.ID
+
+			fmt.Println(string(b))
+
+			// end-create_registration
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(registration).ToNot(BeNil())
+		})
+		It(`GetRegistration request example`, func() {
+			fmt.Println("\nGetRegistration() result:")
+			// begin-get_registration
+
+			getRegistrationOptions := partnerCenterSellService.NewGetRegistrationOptions(
+				registrationId,
+			)
+
+			registration, response, err := partnerCenterSellService.GetRegistration(getRegistrationOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(registration, "", "  ")
+			fmt.Println(string(b))
+
+			// end-get_registration
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(registration).ToNot(BeNil())
+		})
+		It(`UpdateRegistration request example`, func() {
+			fmt.Println("\nUpdateRegistration() result:")
+			// begin-update_registration
+
+			registrationPatchModel := &partnercentersellv1.RegistrationPatch{}
+			registrationPatchModelAsPatch, asPatchErr := registrationPatchModel.AsPatch()
+			Expect(asPatchErr).To(BeNil())
+
+			updateRegistrationOptions := partnerCenterSellService.NewUpdateRegistrationOptions(
+				registrationId,
+				registrationPatchModelAsPatch,
+			)
+
+			registration, response, err := partnerCenterSellService.UpdateRegistration(updateRegistrationOptions)
+			if err != nil {
+				panic(err)
+			}
+			b, _ := json.MarshalIndent(registration, "", "  ")
+			fmt.Println(string(b))
+
+			// end-update_registration
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(200))
+			Expect(registration).ToNot(BeNil())
+		})
+		It(`DeleteRegistration request example`, func() {
+			// begin-delete_registration
+
+			deleteRegistrationOptions := partnerCenterSellService.NewDeleteRegistrationOptions(
+				registrationId,
+			)
+
+			response, err := partnerCenterSellService.DeleteRegistration(deleteRegistrationOptions)
+			if err != nil {
+				panic(err)
+			}
+			if response.StatusCode != 204 {
+				fmt.Printf("\nUnexpected response status code received from DeleteRegistration(): %d\n", response.StatusCode)
+			}
+
+			// end-delete_registration
+
+			Expect(err).To(BeNil())
+			Expect(response.StatusCode).To(Equal(204))
+		})
+		It("Login to account containing a programmatic name approved product", func() {
+			var err error
```

Comment:
I can see that you are constructing a second instance of the service (presumably because it needs different credentials).
I would suggest doing this using a slightly different approach in order to minimize manual work when re-generating the tests and examples...

In the section above where the service is constructed ("Client initialization"), you could define and initialize a second "service" variable (perhaps called `partnerCenterSellServiceAlt` (Alt for alternate) using this code:
```
partnerCenterSellServiceOptionsAlt := &partnercentersellv1.PartnerCenterSellV1Options{
	ServiceName: "partner_center_sell_alt",
}
partnerCenterSellServiceAlt, err = partnercentersellv1.NewPartnerCenterSellV1UsingExternalConfig(partnerCenterSellServiceOptions)
if err != nil {
	panic(err)
}
```
this code fragment would be placed just after the existing code that initializes `partnerCenterSellService`.

Then, move the contents of your second .env file into your primary .env file, but change the property names to reflect the `PARTNER_CENTER_SELL_ALT` service name (instead of `PARTNER_CENTER_SELL`.

Then, for each operation that should be invoked with this alternate service instance, modify the code to use the `partnerCenterSellAlt` service variable.

I just noticed that you also create a third instance of the service to test the badge-related operations... you could extend my suggestion above to include a third instance of the service that is stored in a third variable.

This all assumes that you need to use three different services to invoke the full set of operations.  Of course, if you can use one service instance (perhaps using creds that have been given authority for all three sets of operations) that would be ideal and would be easier to manager over time.